filebeat/harvester/reader/docker_json.go

```diff
@@ -100,6 +100,21 @@ func (p *DockerJSON) Next() (Message, error) {
 
 		if strings.HasPrefix(string(message.Content), "{") {
 			message, err = parseDockerJSONLog(message, &dockerLine)
+			if err != nil {
+				return message, err
+			}
+			// Handle multiline messages, join lines that don't end with \n
+			for message.Content[len(message.Content)-1] != byte('\n') {
```

Comment:
Should we enable this first through a config option and have it as default later on? This would allow us to start playing around with it.

@exekias I'm glad we have the docker prospector ;-)

Reply:
That's a great idea, I'll add it